MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -1030,7 +1041,7 @@ private void initMenu() {
         // The Help menu uses the following Mnemonic keys as of 19-March-2020:
         // A
         JMenu menuHelp = new JMenu(resourceMap.getString("menuHelp.text")); // NOI18N
-        menuHelp.setMnemonic(KeyEvent.VK_H);
+        menuHelp.setMnemonic(KeyEvent.VK_SLASH);
```

Comment:
ALT+H is the standard mnemonic for a help menu.

Reply:
Yeah, I know... but I prioritized based on use, and until our help menu is improved it is not a useful one... although I did just realize G is open for hanGar.

Reply:
Should I make that swap?